index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body class="page__body">
+    <header class="main-header">
+        <!--А можно сделать логотип фоном?-->
+        <a class="main-header__logo-link" href="/">
+            <img class="main-header__logo-image" src="image\mobilelogo.svg" width="33,2px" height="36.6px" alt="логотип cat energy"> </a>
```

Comment:
В пути к файлам должен использоваться обычный слеш, а не обратный. Должно быть так:
```suggestion
            <img class="main-header__logo-image" src="image/mobilelogo.svg" width="33,2px" height="36.6px" alt="логотип cat energy"> </a>
```